assignment-final.Rmd

```diff
@@ -1,5 +1,39 @@
 ---
-title: "Final assignment"
+title: "Final assignment: Scientific report"
+output:
+    html_document:
+        toc: false
 ---
 
-> Uploaded when it is handed out.
+The final assignment is fairly straightforward, but also challenging. You, as a
+team, must write up a scientific report/manuscript. The best way to think about 
+this assignment is to imagine you are a scientist and you are writing a paper
+to publish your research. This is essentially what we expect of you from the
+final report.
+
+So, your report should have at least these sections: "Background and Rationale",
+"Research Objectives and Hypotheses", "Methods" (with "Data Description" and
+"Data Analysis" subsections), "Results", "Discussion", and "Final Conclusion".
+
+For the report, you are expected to:
+
+- Search the previous research and literature on your research questions (this
+doesn't have to be extensive).
```

Comment:
Yes, I really think this should not be emphasized. Sure they have to write some kind of introduction, but they might spend way to long time on this if we are not clear. And I don't think it matters much if their results are not completely novel, as long as they learn the process of extracting and presenting them.